src/current/_includes/releases/v24.1/v24.1.0-rc.2.md

```diff
@@ -0,0 +1,26 @@
+## v24.1.0-rc.2
+
+Release Date: May 16, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-0-rc-2-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that was present since v22.2 where [changefeeds]({% link v24.1/change-data-capture-overview.md %}) with long-running [initial scans]({% link v24.1/create-changefeed.md %}#initial-scan) may incorrectly restore checkpoint job process and drop events during [changefeed restarts]({% link v24.1/changefeed-messages.md %}#duplicate-messages) due to transient errors or node restarts. The bug was most likely to occur in clusters with the following contributing factors:
+    - The `changefeed.shutdown_checkpoint.enabled` [cluster setting]({% link v24.1/cluster-settings.md %}) was enabled (in clusters running v23.2 and later).
+    - The cluster settings `changefeed.frontier_checkpoint_frequency` and `low changefeed.frontier_highwater_lag_checkpoint_threshold` were set low, which result in the initial scan taking many multiples of the configured frequency to complete.
+    - There were multiple target tables with significant differences in row counts in one changefeed.
+    - The changefeed target tables were large with many ranges.
+    - The initial scan took a long time to complete (an hour or longer). [#123968][#123968]
+- Fixed a crash introduced in v24.1.0-beta.2 that could occur when planning [statistics collection]({% link v24.1/cost-based-optimizer.md %}#table-statistics) on a table with a [virtual computed column]({% link v24.1/computed-columns.md %}) using a user-defined type and the `sql.stats.virtual_computed_columns.enabled` [cluster setting]({% link v24.1/cluster-settings.md %}) is set to `true`. `sql.stats.virtual_computed_columns.enabled` was introduced in v24.1.0-alpha.1 by default enabled. [#124064][#124064]
+
+<div class="release-note-contributors" markdown="1">
+
+<h3 id="v24-1-0-rc-2-contributors">Contributors</h3>
+
+This release includes 3 merged PRs by 3 authors.
+
+</div>
```

Comment:
I believe I've removed this in the past when there are no community contributions. May be too awkward/confusing to leave it in when there are only 2 linked PRs here? I could also understand leaving it, if readers have grown accustomed to having a count.